docs/en/query_language/alter.md

```diff
@@ -68,9 +68,28 @@ For tables that don't store data themselves (such as `Merge` and `Distributed`),
 
 The `ALTER` query for changing columns is replicated. The instructions are saved in ZooKeeper, then each replica applies them. All `ALTER` queries are run in the same order. The query waits for the appropriate actions to be completed on the other replicas. However, a query to change columns in a replicated table can be interrupted, and all actions will be performed asynchronously.
 
+<a name="query_language-queries-alter-key_alters"></a>
+
+### Manipulations With Key Expressions
+
+The following command is supported:
+
+``` sql
+MODIFY ORDER BY new_expression
+```
+
+It only works for tables in the `MergeTree` family (including replicated tables). The command changes the
+[sorting key](../operations/table_engines/mergetree.md#table_engines-mergetree-sorting_key) of the table
+to `new_expression` (an expression or a tuple of expressions). Primary key remains the same.
+
+The command is lightweight in a sense that it only changes metadata. To keep the property that data part
+rows are sorted by the sorting key expression you cannot add expressions containing existing columns
```

Comment:
"sorted by the sorting..." sounds bad, "ordered by the sorting..." would be better I think